tests/cli/test_compose_sanity.sh

```diff
@@ -22,6 +22,26 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        if [ -n "$UUID" ]; then
+            until $CLI compose info $UUID | grep 'RUNNING'; do
```

Comment:
There is some possibility that this could return FAILED and then it would be stuck in the loop.

Reply:
Thanks for noticing this. I added a check for FAILED compose. I guess this should go to master branch too.